frontend/src/theme/globalStyles.ts

```diff
@@ -70,7 +70,13 @@ export const globalStyles = (theme: Theme): any => css`
     color: ${theme.colors.disabled};
   }
 
-  // VegaLite-specific CSS to style tootips
+  // glide-data-grid
+  // Fix issue with empty option in dropdown cell
+  div#portal div.click-outside-ignore > div > div {
+    min-height: ${theme.spacing.threeXLPx}px;
+  }
```

Comment:
This makes me cry. Is this in the data grid itself? Can we point to an issue? I just worry about the brittleness of a dependency update.